src/update-done/update-done.c

```diff
@@ -84,6 +95,12 @@ static int parse_argv(int argc, char *argv[]) {
                 case 'h':
                         return help();
 
+                case ARG_ROOT:
+                        r = parse_path_argument(optarg, false, &arg_root);
```

Comment:
2nd argument should be prefixed with `/* suppress_root= */`. And it actually should be set as true